filecoin-proofs/src/bin/paramfetch.rs

```diff
@@ -102,6 +102,19 @@ Defaults to '{}'
                 .long("ipget-bin")
                 .help("Use specific ipget binary instead of looking for (or installing) one in /var/tmp/ipget/ipget"),
         )
+        .arg(
+            Arg::with_name("ipget-args")
+                .takes_value(true)
+                .long("ipget-args")
+                .help("Specify additional arguments for ipget")
+        )
+        .arg(
+            Arg::with_name("ipget-version")
```

Comment:
Blocking: This argument cannot be combined with `ipget-bin`. Please do use `conflicts_with`.